spk/adminer/src/.htaccess

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Please rename source file as `dot-htaccess` for instance to prevent it to be hidden in Git working copy.